sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/util/CosmosPagedFlux.java

```diff
@@ -36,10 +37,29 @@
 
     private final Function<CosmosPagedFluxOptions, Flux<FeedResponse<T>>> optionsFluxFunction;
 
+    private Consumer<FeedResponse<T>> feedResponseConsumer;
```

Comment:
```suggestion
    private final Consumer<FeedResponse<T>> feedResponseConsumer;
```

Reply:
It can't be final as this is optional. We only set this in 1 constructor which is invoked by `handle()` API.

Reply:
you can set it to null in the other constructor.

Reply:
Yes, that is possible, we can do that.